### PR TITLE
app-backup/kfoldersync: New package, bug 507500

### PR DESCRIPTION
**Pull request by a17r**
Based on work done by Frank Krömmelbein and Lorenzo Porta (Vindex17)
See also https://bugs.gentoo.org/show_bug.cgi?id=507500

Package-Manager: portage-2.2.24

### CODE REVIEW

**Comment by Vindex17** [comment]
Why these?
> $(add_frameworks_dep kconfig)
> $(add_frameworks_dep kconfigwidgets)
> $(add_frameworks_dep kitemviews)
> $(add_frameworks_dep kwidgetsaddons)
> dev-qt/qtgui:5

**Comment by a17r** [comment]
kfoldersync binary ends up linking with them.

**Comment by Vindex17** [comment]
Because they are mandatory dependencies of the others, in fact I didn't even put them in CMakeLists.txt

File: app-backup/kfoldersync/kfoldersync-3.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+KDE_GCC_MINIMAL="4.9"
```
Comment (Vindex17) [inline app-backup/kfoldersync/kfoldersync-3.0.0.ebuild]: Thank you for pointing that out. I think I'll add a gcc version check even in CMakeLists.txt in the next release

**Comment by kensington** [comment]
In fact all of those dependencies are directly used - check the header includes.

**Comment by kensington** [comment]
Thanks, rebased as d3aff3cf218f5b259e9891473f4c189e7168daf2.

**Pull request closed without merging**